### PR TITLE
Add ANSI logo to codemon

### PR DESCRIPTION
**Pull request by alphaX86**
**PR Details**
- This PR includes a new ANSI-based logo for the codemon project
- The logo appears every time during `codemon` and `codemon --help`
- The font used is **Small Slant** and it's FIGlet based
- This solves Issue #53 
- Any suggestions for logo or font need to be changed?? Refer this [link] for more FIGlet based fonts for suggestion(https://manytools.org/hacker-tools/ascii-banner/)
- Any else? Nope 😆

### CODE REVIEW

**Comment by ankushbhardwxj** [comment]
@alphaX86 - This logo doesn't match to the one that you sent on gitter. Please make necessary changes.

**Comment by alphaX86** [comment]
@ankingcodes - Changes are made

File: codemon/CodemonHelp.py [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
-/ /__/ /_/ / // / _// /|_/ / /_/ /    / 
-\___/\____/____/___/_/  /_/\____/_/|_/  
-                                        """))
+  print(colored.green(""" ██████╗ ██████╗ ██████╗ ███████╗███╗   ███╗ ██████╗ ███╗   ██╗
+██╔════╝██╔═══██╗██╔══██╗██╔════╝████╗ ████║██╔═══██╗████╗  ██║
+██║     ██║   ██║██║  ██║█████╗  ██╔████╔██║██║   ██║██╔██╗ ██║
+██║     ██║   ██║██║  ██║██╔══╝  ██║╚██╔╝██║██║   ██║██║╚██╗██║
+╚██████╗╚██████╔╝██████╔╝███████╗██║ ╚═╝ ██║╚██████╔╝██║ ╚████║
+ ╚═════╝ ╚═════╝ ╚═════╝ ╚══════╝╚═╝     ╚═╝ ╚═════╝ ╚═╝  ╚═══╝
+                                                               """))
```
Comment (ankushbhardwxj) [inline codemon/CodemonHelp.py]: Can you add one newline here ?

File: codemon/CodemonHelp.py [hunk 2]
```diff
@@ -1,11 +1,13 @@
 from clint.textui import colored
 
 def showHelp():
```
Comment (ankushbhardwxj) [inline codemon/CodemonHelp.py]: Can you add two newlines right after the function definition ?

**Review by ankushbhardwxj — CHANGES_REQUESTED** [review]

**Comment by ankushbhardwxj** [comment]
```
$ codemon


 ██████╗ ██████╗ ██████╗ ███████╗███╗   ███╗ ██████╗ ███╗   ██╗
██╔════╝██╔═══██╗██╔══██╗██╔════╝████╗ ████║██╔═══██╗████╗  ██║
██║     ██║   ██║██║  ██║█████╗  ██╔████╔██║██║   ██║██╔██╗ ██║
██║     ██║   ██║██║  ██║██╔══╝  ██║╚██╔╝██║██║   ██║██║╚██╗██║
╚██████╗╚██████╔╝██████╔╝███████╗██║ ╚═╝ ██║╚██████╔╝██║ ╚████║
 ╚═════╝ ╚═════╝ ╚═════╝ ╚══════╝╚═╝     ╚═╝ ╚═════╝ ╚═╝  ╚═══╝


A CLI tool to ace competitive programming contests

COMMANDS:

codemon - - - - - - - - - - - - - - -  displays this message
codemon --help - - - - - - - - - - - - shows help
codemon init <contestName> - - - - - - initialises a contest
codemon init -n <file> - - - - - - - - creates file with given name
codemon listen - - - - - - - - - - - - compiles and gives output
codemon reg - - - - - - - - - - - - -  Register user details
codemon practice <dirName> - - - - - - Starts a practice session
```
@paramsiddharth @Enigmage - this is how it looks right now. Looks good to me, what do y'all think ?

**Comment by ankushbhardwxj** [comment]
@alphaX86 - Also, remember to  update your branch and you may also try to fix #67 here itself ?

File: codemon/CodemonHelp.py (hunk 2 above)
Comment (alphaX86) [reply codemon/CodemonHelp.py]: Right after function definition?? Why is that so?

**Comment by alphaX86** [comment]
> @alphaX86 - Also, remember to update your branch and you may also try to fix #67 here itself ?

I accept this!

File: codemon/CodemonHelp.py (hunk 1 above)
Comment (alphaX86) [reply codemon/CodemonHelp.py]: At line no?

File: codemon/CodemonHelp.py (hunk 2 above)
Comment (ankushbhardwxj) [reply codemon/CodemonHelp.py]: @alphaX86 - basically I want two newlines above the logo and one newline after the logo

**Comment by paramsiddharth** [comment]
> 
> @paramsiddharth @Enigmage - this is how it looks right now. Looks good to me, what do y'all think ?

image
Looks amazing inside the terminal!

**Comment by ankushbhardwxj** [comment]
@alphaX86 - can you add two newlines before the logo and one newline after the logo, so that I can go ahead and merge this ?

**Comment by alphaX86** [comment]
> @alphaX86 - can you add two newlines before the logo and one newline after the logo, so that I can go ahead and merge this ?

Stand by @ankingcodes I'll give a green light when complete

**Comment by alphaX86** [comment]
Do you need colors @ankingcodes ?

**Comment by alphaX86** [comment]
image
This is how it looks now

**Comment by ankushbhardwxj** [comment]
@alphaX86 - please keep it to the previous version. Coloured logo doesn't look good for users who use themes for their terminals.

**Comment by alphaX86** [comment]
> @alphaX86 - please keep it to the previous version. Coloured logo doesn't look good for users who use themes for their terminals.

@ankingcodes To be honest, someone already added green color to the terminal in previous version and I didn't put that one :(

**Comment by alphaX86** [comment]
Well, I'll remove it ASAP

**Comment by alphaX86** [comment]
It's removed

**Pull request merged**